internal/pkg/yamlutils/yamlutils_test.go

```diff
@@ -13,21 +13,71 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package yamlutils
+package yamlutils_test
```

Comment:
why?

Reply:
I'm in favor of the style described e.g. in [this great post](https://medium.com/@matryer/5-simple-tips-and-tricks-for-writing-unit-tests-in-golang-619653f90742#:~:text=Put%20your%20tests%20in%20a,real%20user%20of%20the%20package.)

Reply:
I get the premise now, It's actually an interesting point, I think I agree too :)
there's one thing from that post which boggles me though: "Run all tests on save" :D either this guy is working on very small projects or he's working on a supercomputer

Reply:
Hehe, I think `oslo` fulfills the definition of a small project ;)